content/services/classroom/featured-carousel.yaml

```
@@ -1,38 +1,38 @@
 carousel:
   - title: "Introduction to Linux"
-    category: "Linux"
+    category: ["Linux"]
```

then all of these need to be renamed category --> categories

```suggestion
    categories: ["Linux"]
```